apps/document-storage/src/server/app/routes.js

```diff
@@ -1,21 +1,80 @@
 import express from 'express';
-import { asyncHandler } from './middleware/async-handler.js';
-import { validateDocumentUpload, validateGetAllDocuments, validateUploadDocument, validateDocumentName } from './validator.js';
 import { downloadDocument, getAllDocuments, uploadDocument } from './controller.js';
+import { asyncHandler } from './middleware/async-handler.js';
+import { validateDocumentName,validateGetAllDocuments, validateUploadDocument } from './validator.js';
 
-const router = express.Router();
+const router = new express.Router();
 
 router.get('/', 
+	/*
```

Comment:
is this commented out code still necessary? - if so can you add a comment please

Reply:
this is what generates the api documentation in swagger. so we need this in there